tesseract/tesseract_planning/tesseract_process_managers/examples/freespace_manager_example.cpp

```diff
@@ -62,47 +61,35 @@ int main()
   if (plotter != nullptr)
   {
     plotter->init(tesseract);
-    plotter->waitForConnection();
-    plotter->plotEnvironment();
+    plotter->waitForConnection(3);
+    if (plotter->isConnected())
+      plotter->plotEnvironment();
   }
 
-  // --------------------
+  // Create Process Planning Server
+  ProcessPlanningServer planning_server(std::make_shared<ProcessEnvironmentCache>(tesseract), 1);
+  planning_server.loadDefaultProcessPlanners();
+
+  // Create Process Planning Request
+  ProcessPlanningRequest request;
+  request.name = process_planner_names::FREESPACE_PLANNER_NAME;
```

Comment:
Perhaps show how to modify a profile?

Reply:
I will include this in a follow on PR. After this is merged I will move on to finishing the examples.